bedrock/security/management/commands/update_security_advisories.py

```diff
@@ -65,7 +65,7 @@ def add_or_update_advisory(data, html):
     :return: SecurityAdvisory
     """
     mfsa_id = data.pop("mfsa_id")
-    year, order = [int(x) for x in mfsa_id.split("-")]
+    year, order = (int(x) for x in mfsa_id.split("-"))
```

Comment:
**question (non-blocking):** Is there much to be gained by making this a generator, given we immediately exhaust it?

Reply:
pyupgrade did it so I wasn't going to undo it. My feeling was to leave it as it's probably generally better and it's a good pattern to follow? Open to other opinions though.